be/src/formats/parquet/page_reader.h

```diff
@@ -114,6 +114,7 @@ class PageReader {
     BufferPtr _compressed_buf;
     BufferPtr _uncompressed_buf;
     BufferPtr _cache_buf;
+    bool _hit_cache;
```

Comment:
Why not set default value for `_hit_cache`?

Reply:
confirmed